certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+                "type": "string"
+              },
+              "\"vary\":": {
+                "description": "\"vary\":",
+                "type": "string"
+              },
+              "\"x-aspnet-version\":": {
+                "description": "\"x-aspnet-version\":",
+                "type": "string"
+              },
+              "\"x-frame-options\":": {
+                "description": "\"x-frame-options\":",
+                "type": "string"
+              },
+              "\"x-powered-by\":": {
+                "description": "\"x-powered-by\":",
+                "type": "string"
+              },
+              "}": {
+                "description": "}",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Add Individual",
+        "description": "Adds an Individual",
+        "operationId": "Add-Individual",
+        "parameters": [
+          {
+            "name": "createUser",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "False",
+            "description": "Choose True if a user account should be created",
+            "enum": [
+              "True",
+              "False"
+            ],
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "Creates User."
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "firstName": {
+                  "type": "string",
+                  "description": "firstName"
+                },
+                "lastName": {
+                  "type": "string",
+                  "description": "lastName"
+                },
+                "email": {
+                  "type": "string",
+                  "description": "email",
+                  "title": "",
+                  "x-ms-visibility": "Advanced"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Individuals/{id}/Activities": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.